components/logging/org.wso2.carbon.logging.updater/pom.xml

```diff
@@ -14,41 +14,40 @@
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
 -->
+
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 
     <parent>
         <groupId>org.wso2.carbon.commons</groupId>
         <artifactId>logging</artifactId>
         <version>4.7.2-SNAPSHOT</version>
-	<relativePath>../pom.xml</relativePath>
+        <relativePath>../pom.xml</relativePath>
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>org.wso2.carbon.logging.view.ui</artifactId>
+    <artifactId>org.wso2.carbon.logging.updater</artifactId>
     <packaging>bundle</packaging>
-    <name>WSO2 Carbon - Logging View UI</name>
+    <name>WSO2 Carbon - logging config updater</name>
     <url>http://wso2.org</url>
-    <description>Log Viewer UI Bundle</description>
+
     <build>
         <plugins>
             <plugin>
                 <groupId>org.apache.felix</groupId>
                 <artifactId>maven-bundle-plugin</artifactId>
-                <version>1.4.0</version> 
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
-                        <Export-Package>
-                            org.wso2.carbon.logging.view.ui.*
-                        </Export-Package>
-			<Import-Package>
-                            org.wso2.carbon.logging.view.stub.*; version="${carbon.commons.imp.package.version}",
+                        <Private-Package>org.wso2.carbon.logging.updater.internal</Private-Package>
+                        <Import-Package>
+                            !org.wso2.carbon.logging.updater.*,
+                            org.wso2.carbon.utils.*;version="${carbon.kernel.imp.pkg.version}",
+                            org.osgi.service.cm.*,
                             *;resolution:=optional
-			</Import-Package>
-			<!--Embed-Dependency>org.wso2.carbon.logging.view.stub;scope=compile|runtime;inline=true</Embed-Dependency-->
-                        <Carbon-Component>UIBundle</Carbon-Component>
+                        </Import-Package>
+                        <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
Can we remove this if there is no requirement?

Reply:
Fixed with https://github.com/wso2/carbon-commons/pull/369